node/pkg/accountant/query_test.go

```diff
@@ -145,3 +149,84 @@ func TestParseBatchTransferStatusPendingResponse(t *testing.T) {
 	// Use DeepEqual() because the response contains pointers.
 	assert.True(t, reflect.DeepEqual(expectedResult, response.Details[0]))
 }
+
+// BatchTransferStatusQueryConnMock allows us to mock batch_transfer_status by implementing SubmitQuery.
+type BatchTransferStatusQueryConnMock struct {
+	resp []byte
+}
+
+func (qc *BatchTransferStatusQueryConnMock) SubmitQuery(ctx context.Context, contractAddress string, query []byte) ([]byte, error) {
+	if len(query) > 150*maxPendingsPerQuery {
```

Comment:
would be best to leave a comment here to explain the 150

Reply:
Added a comment.